apps/teams-test-app/e2e-test-data/dialogCard.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "DialogCard",
```

Comment:
Isn't it the sub capability of capability? If we are following the naming as page, i.e., pages.config, it needs to be Dialog Card. Same to others